packages/calypso-e2e/src/lib/pages/plugins-page.ts

```diff
@@ -71,4 +78,48 @@ export class PluginsPage {
 		await this.page.click( selectors.breadcrumb( text ) );
 		assert.match( this.page.url(), /.*\/plugins$/ );
 	}
+
+	/**
+	 * Click Monthly Pricing
+	 */
+	async clickMonthlyPricing(): Promise< void > {
```

Comment:
This is a convention we need to document, so our bad on this, but if you end up needing the widescreen toggle tests, I would recommend abstracting away the viewport size handling from the test scripts as much as possible by including branching logic based on viewport size within the methods on the POM class here.

So, e.g., just have two methods `selectMonthlyPricing` and `selectAnnualPricing` and within those check if it's mobile or desktop and use the right selector based on that ENV variable value.